File: LeetCode/9_Palindrome_Number.py
def isPalindrome(x):
    if x < 0:
        return False
    
    z = x
    rev = 0
    while x > 0:
        rev = rev*10 + (x%10)
        x = int(x/10)
    return z == rev



def isPalindrome(x):
    if x < 0:
        return False
    z = x 
    rev = 0
    while z > 0:
        rev = rev*10 + (z%10)
        z = z//10
    print(x, rev, z)
    return x == rev
x = 121
x = 123454321
# x = -123454321
# Output: true
print(isPalindrome(x))

File: LeetCode/9_Palindrome_Number.js
var isPalindrome = function(x) {
    return x == x.toString().split('').reverse().join('')
};

var isPalindrome = function(x) {
    if (x < 0) {
        return false
    }
    var z = x
    let rev = 0
    while (x>0) {
        rev = rev*10 + (x%10);
        x = parseInt(x/10);
    }
    return z == rev
}
x = 12321
// Output: true


console.log(isPalindrome(x))
